cmake/Config.cmake.in

```diff
@@ -2,13 +2,13 @@
 
 include(CMakeFindDependencyMacro)
 
-find_dependency(OpenCV REQUIRED)
-find_dependency(Boost CONFIG REQUIRED)
-find_dependency(Eigen3 CONFIG REQUIRED)
-find_dependency(cereal CONFIG REQUIRED)
-find_dependency(glm REQUIRED)
-find_dependency(nanoflann CONFIG REQUIRED)
-find_dependency(eigen3-nnls CONFIG REQUIRED)
+find_dependency(OpenCV)
+find_dependency(Boost CONFIG system)
```

Comment:
I'm not sure that this will work too. It's easier to use `find_package` instead, in this case we can use `REQUIRED`.